packages/core/snapshot/read.go

```diff
@@ -9,18 +9,13 @@ import (
 	"io"
 
 	"github.com/cockroachdb/errors"
-	"github.com/iotaledger/hive.go/core/serix"
-
 	"github.com/iotaledger/goshimmer/packages/core/epoch"
 	"github.com/iotaledger/goshimmer/packages/core/ledger"
+	"github.com/iotaledger/hive.go/core/serix"
 )
 
 // streamSnapshotDataFrom consumes a snapshot from the given reader.
-func streamSnapshotDataFrom(
-	reader io.ReadSeeker,
-	headerConsumer HeaderConsumerFunc,
-	outputConsumer UTXOStatesConsumerFunc,
-	epochDiffsConsumer EpochDiffsConsumerFunc) error {
+func streamSnapshotDataFrom(reader io.ReadSeeker, headerConsumer HeaderConsumerFunc, outputConsumer UTXOStatesConsumerFunc, epochDiffsConsumer EpochDiffsConsumerFunc, activityLogConsumer ActivityLogConsumerFunc) error {
```

Comment:
Could you align it as before? :stuck_out_tongue:

Reply:
It looks like some of my format settings like to straighten this up :smile: